packages/oc-template-react-compiler/lib/viewTemplate.js

```diff
@@ -20,7 +22,7 @@ const viewTemplate = ({
           '["oc", "reactComponents", "${bundleHash}"],' + 
           '"' + staticPath + '${bundleName}.js",' +
           'function(ReactComponent){' +
-            'var targetNode = document.getElementById("${reactRoot}");' +
+            'var targetNode = document.getElementById("'+ reactRoot +'");' +
```

Comment:
^ Follows previous suggestions:
```suggestion
            'var targetNode = document.getElementById("'+ reactRootId +'");' +
```